### DISCUSSION TITLE
Running GraphQL on Lambda with Rust

### DISCUSSION THREAD

**Discussion by dbanty** [dbanty/dylananthony.com#56] [Posts]
The discussion for [this post](https://dylananthony.com/posts/graphql-lambda-rust) which is a high level overview of writing a GraphQL API in Rust and hosting it on AWS Lambda behind API Gateway.

I'm considering writing a full tutorial on this topic with things like auth, websocket subscriptions, database migrations... everything you need. I'd likely charge for the full thing given the amount of work it'll be, but I'll release snippets as blog posts while I'm working on it. If that's something you'd want to see make sure to let me know down below!

**Comment by heliumbrain** [comment]
I would defnitely be interested in seeing (and paying for) something like that :) I've been following your blog posts, and learned a lot from it. Also came from using FastAPI quite a lot and now wanting to learn Rust.
